examples/lib/tour/01-getting-started/README.md

```diff
@@ -3,11 +3,265 @@
 This [tour](example.go) illustrates the basic usage of the API to
 access component versions in an OCM repository.
 
+## Running the example
+
 You can just call the main program with some config file argument
 with the following content:
 
 ```yaml
 component: github.com/mandelsoft/examples/cred1
 repository: ghcr.io/mandelsoft/ocm
 version: 0.1.0
-```
\ No newline at end of file
+```
+
+## Walkthrough
+
+The basic entry point for using the OCM library is always
+an [OCM Context object](../../contexts.md). It bundles all
+configuration settings and type registrations, like
+access methods, repository types, etc, and
+configuration settings, like credentials,
+which should be used when working with the OCM
+ecosystem.
+
+Therefore, the first step is always to get access to such
```

Comment:
```suggestion
The first step is always to get access to such
```

Reply:
*Therefore* relates to the paragraph before, therefore I think it should be ok.